### PR TITLE
[BUG] fix metrics bug

### PR DESCRIPTION
**Pull request by kpango**
Signed-off-by: kpango <kpango@vdaas.org>

### Description:
v1.6.0 includes rpc status metrics bug, when RPC call succeeds metrics expose the state as Unknown it should be OK Status, so I changed it.

### Environment:

- Go Version: 1.19.1
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 1.14.8

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/main/CONTRIBUTING.md) document.
- [ ] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by hlts2 — APPROVED** [review]
Thank you 🙏 
LGTM

File: internal/net/grpc/interceptor/server/metric/metric.go [hunk 1]
```diff
@@ -54,51 +54,32 @@ func MetricInterceptors() (grpc.UnaryServerInterceptor, grpc.StreamServerInterce
 		return nil, nil, errors.Wrap(err, "failed to create completedRPCs metric")
 	}
 
+	record := func(ctx context.Context, method string, err error, latency float64) {
+		code := codes.OK // default error is success when error is nil
+		if err != nil {
+			st, _ := status.FromError(err)
+			if st != nil {
+				code = st.Code()
+			}
```
Comment (kevindiu) [inline internal/net/grpc/interceptor/server/metric/metric.go:63]: do you think we need to handle the case that if the status cannot be parsed?
```suggestion
			if st != nil {
				code = st.Code()
			} else {
				code = codes.Unknown
			}
```

Comment (kpango) [reply internal/net/grpc/interceptor/server/metric/metric.go:63]: status.FromError returns Unknown status if parse failure.
we don't need handle it.

Comment (kpango) [reply internal/net/grpc/interceptor/server/metric/metric.go:63]: FYI: https://github.com/vdaas/vald/blob/main/internal/net/grpc/status/status.go#L170

**Review by kevindiu — APPROVED** [review]

**Pull request merged**